### PR TITLE
Release notes for OpenJ9 0.24.0

### PR DESCRIPTION
**Pull request by SueChaplain**
- Updates and changes for the 0.24 release

[ci skip]

Signed-off-by: SueChaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

File: doc/release-notes/0.24/0.24.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Version / Platform</th>
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+
+<tr><td valign="top">
+<a href="https://github.com/eclipse/openj9/pull/10873">#10873</a> and <a href="https://github.com/eclipse/openj9/pull/10082">#11082</a></td>
```
Comment (pshipton) [inline doc/release-notes/0.24/0.24.md:59]: The link is to 10082 and not 11082

File: doc/release-notes/0.24/0.24.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+<td valign="top"><tt>-XX:[+|-]PortableSharedCache</tt> option support update
+</td>
+<td valign="top">All versions</td>
+<td valign="top">This option is now supported on IBM Z systems&reg; and POWER&reg; systems.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/10819">#10819</a></td>
+<td valign="top"><tt>-XX:[+|-]ShareAnonymousClasses</tt> option behavior update</td>
+<td valign="top">OpenJDK 15</td>
+<td valign="top">In earlier releases, this option controls the storage of VM anonymous classes in the shared classes cache. This option now enables and disables the storage of hidden classes in the shared classes cache.</td>
```
Comment (pshipton) [inline doc/release-notes/0.24/0.24.md:81]: I know 15 is in the version column, but I think it should be mentioned in the impact as well that the change is for Java 15+.

File: doc/release-notes/0.24/0.24.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/10774">#10774</a></td>
+<td valign="top">Additional parameters for <strong>jcmd Dump.*</strong> commands</td>
+<td valign="top">All versions</td>
+<td valign="top">Additional parameters can now be included for the <strong>jcmd Dump.*</strong> commands. For more information, see the <a href="https://www.eclipse.org/openj9/docs/version0.24/"> user documentation</a>.</td>
```
Comment (pshipton) [inline doc/release-notes/0.24/0.24.md:87]: The default options for Dump.system and Dump.heap are changed as well to use `request=exclusive+prepwalk`

**Comment by SueChaplain** [comment]
Think it's all ready now @pshipton

File: doc/release-notes/0.24/0.24.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/10774">#10774</a></td>
+<td valign="top">Additional parameters for <tt>jcmd Dump.*</tt> commands</td>
+<td valign="top">All versions</td>
+<td valign="top">Additional parameters can now be included for the <tt>jcmd Dump.*</tt> commands. The default for both system and heap dumps is now `request=exclusive+prepwalk`. For more information, see the <a href="https://www.eclipse.org/openj9/docs/version0.24/"> user documentation</a>.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/10761">#10761</a></td>
+<td valign="top">Change in behavior for the <tt>jextract</tt> utility</td>
+<td valign="top">All versions</td>
+<td valign="top">If the build ID of the <tt>jextract</tt> utility does not match the build ID of the SDK that is recorded in the system dump, an exception message is generated. A new option is introduced to force the utility to continue. </td>
```
Comment (pshipton) [inline doc/release-notes/0.24/0.24.md:93]: This can mention the option is `-r`.

File: doc/release-notes/0.24/0.24.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/10819">#10819</a></td>
+<td valign="top"><tt>-XX:[+|-]ShareAnonymousClasses</tt> option behavior update</td>
+<td valign="top">OpenJDK 15</td>
+<td valign="top">In earlier releases, this option controls the storage of VM anonymous classes in the shared classes cache for all OpenJDK versions. In this release, the option now enables and disables the storage of hidden classes in the shared classes cache on OpenJDK 15 and later versions.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/10774">#10774</a></td>
+<td valign="top">Additional parameters for <tt>jcmd Dump.*</tt> commands</td>
+<td valign="top">All versions</td>
+<td valign="top">Additional parameters can now be included for the <tt>jcmd Dump.*</tt> commands. The default for both system and heap dumps is now `request=exclusive+prepwalk`. For more information, see the <a href="https://www.eclipse.org/openj9/docs/version0.24/"> user documentation</a>.</td>
```
Comment (pshipton) [inline doc/release-notes/0.24/0.24.md:87]: The back quotes around request=exclusive+prepwalk don't do anything to the formatting.

Comment (SueChaplain) [reply doc/release-notes/0.24/0.24.md:87]: Ah, I'll fix that. It's inside HTML.

**Review by pshipton — APPROVED** [review]

**Pull request merged**